src/Mapper/Mapper.test.ts

```diff
@@ -149,6 +150,74 @@ describe("Mapper", () => {
       fakeModel.fields.fakeItems.serialize = false;
     });
 
+    describe("transformPatch", () => {
+      let originalModel;
+      let originalModelSpy;
+      let originalFullText;
+
+      beforeEach(() => {
+        originalFullText = lorem.slug();
+        originalModel = new MockModel({ ...mockModelData, fullText: originalFullText });
+
+        // can't directly use stub because property doesn't exist yet and defining it and then stubbing it doesn't work either.
```

Comment:
I figure we will be stubbing this method anyway for our tests so waiting for https://jira.rediker.com/browse/IO-1907 is not necessary, what do you guys think?

Reply:
I'll remove the comment on line 162 as well because it wont be timeless